Mathlib/Order/OmegaCompletePartialOrder.lean

```diff
@@ -531,6 +531,15 @@ instance (priority := 100) [CompleteLattice α] : OmegaCompletePartialOrder α w
 
 variable [OmegaCompletePartialOrder α] [CompleteLattice β] {f g : α → β}
 
+lemma ωScottContinuous.mediatingMorphism (hf : ωScottContinuous f) (hg : ωScottContinuous g) :
+    ωScottContinuous fun x => (f x, g x) := by
+  refine ωScottContinuous.of_monotone_map_ωSup
```

Comment:
```suggestion
  refine .of_monotone_map_ωSup
```
(maybe it doesn't work yet, it needs a Lean 4 fix that might not have landed yet)

Reply:
Indeed it doesn't work yet - but I've now deduced this lemma from the general case.